src/Symfony/Installer/NewCommand.php

```diff
@@ -350,6 +350,18 @@ protected function updateComposerJson()
             unset($contents['description']);
         }
 
+        if (isset($contents['config']['platform']['php'])) {
+            unset($contents['config']['platform']['php']);
+
+            if (empty($contents['config']['platform'])) {
+                unset($contents['config']['platform']);
+            }
+
+            if (empty($contents['config'])) {
```

Comment:
What about to make it nested to the previous if statement? The `$contents['config']` could be empty only when we unset `$contents['config']['platform']`, otherwise it probably has at least one item.

```php
            if (empty($contents['config']['platform'])) {
                unset($contents['config']['platform']);

                if (empty($contents['config'])) {
                    unset($contents['config']);
                }
            }
```